src/Model/Table/RequestsTable.php

```diff
@@ -131,7 +141,26 @@ public function gc(): void
 
             $conn = $this->getConnection();
             if ($conn->getDriver() instanceof Sqlite) {
-                $conn->execute('VACUUM;');
+                $conn->execute('
+                    PRAGMA auto_vacuum = FULL;
+                    PRAGMA journal_mode = OFF;
+                    PRAGMA synchronous = OFF;
+                    PRAGMA foreign_keys = OFF;
+                    PRAGMA temp_store = MEMORY;
+                    PRAGMA automatic_index = OFF;
+                ');
```

Comment:
Should this only happen if the GC is going to occur?

Reply:
I think it is not needed to run everytime, so it is run when gc is ran